src/backup_camera.cpp

```diff
@@ -146,14 +154,33 @@ void BackupCamera::processGPIO()
 void BackupCamera::signalToQuit()
 {
     camera_one_->signalToQuit();
-    //song_player_one_->songQuit();
+
 }
 
 void BackupCamera::close()
 {
-    //song_player_one_->closeSongPlayer();
-    //song_player_one_->WaitForThreadToExit();
+
     camera_one_->WaitForThreadToExit();
 }
 
+void BackupCamera::resizeCameraRect(SDL_Window* window, bool setFullscreenNext)
+{
+    int w, h;
+    SDL_GetWindowSize(window, &w, &h);
+    SDL_Rect camera_one_new_rect;
+    camera_one_new_rect.x = 0;
+    camera_one_new_rect.y = 0;
 
+    if (setFullscreenNext) //should work witthout the if statement, but moving back to window doesnt shrink back for some reason when reading window size
```

Comment:
This "setFullScreenNext" variable is poorly named, just naming it `fullscreen` would suffice and be more clear.